src/lib.rs

```diff
@@ -0,0 +1,26 @@
+#![no_std]
+
+// Re-export the correct HAL based on which feature is active
+#[cfg(feature = "esp32")]
+pub use esp32_hal as hal;
+#[cfg(feature = "esp32c3")]
+pub use esp32c3_hal as hal;
+#[cfg(feature = "esp32s3")]
+pub use esp32s3_hal as hal;
+// Due to a bug in esp-hal this MUST be included in the root.
+#[cfg(feature = "esp32c3")]
+pub use hal::interrupt;
```

Comment:
Which bug is this? Is there an issue?

Reply:
It's referring to https://github.com/esp-rs/esp-hal/issues/197